meta-dts-distro/recipes-dts/dts/dts/dts-environment.sh

```diff
@@ -6,6 +6,7 @@ BOARD_MODEL="$(dmidecode -s baseboard-product-name)"
 
 BIOS_VENDOR="$(dmidecode -s bios-vendor)"
 BIOS_VERSION="$(dmidecode -s bios-version)"
+DASHARO_VERSION="$(echo $BIOS_VERSION | cut -d ' ' -f 3 | tr -d 'v')"
```

Comment:
For Dasharo fw used on optiplex
```
# dmidecode -s bios-version
Dasharo (coreboot+SeaBIOS)
```

So DASHARO_VERSION is ""

Reply:
I guess we need to fix optiplex

Reply:
for sure

Reply:
version syntax added https://github.com/Dasharo/meta-dts/pull/29/commits/29fd7b1f4d992c080e398732a27ed147a6fc0953#diff-ac1fdd7af29e82bf8d7ca23462952d638564fce3189e36cbb6eebeb48f480d20R630 so in such cases we should know if something bad happened